.github/scripts/simpleauth-azdo-pipelines.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+countNum=1
+restUrl="https://dev.azure.com/mseng/VSIoT/_apis/build/latest/$2?api-version=6.0-preview.1"
+rsp=$(curl -u :$1 $restUrl | jq -r '.value| .[0]')
+status=$(echo $rsp | jq -r '.state')
+buildId=$(echo $rsp | jq -r '.id')
```

Comment:
this step is get latest build pipeline id and status, if status is completed, we can get asset url leverage the build id